secretstores/hashicorp/vault/vault.go

```diff
@@ -176,8 +176,8 @@ func (v *vaultSecretStore) getSecret(secret string) (*vaultKVResponse, error) {
 		var b bytes.Buffer
 		io.Copy(&b, httpresp.Body)
 
-		return nil, fmt.Errorf("couldn't to get successful response: %#v, %s",
-			httpresp, b.String())
+		return nil, fmt.Errorf("couldn't get successful response, status code %d, body %s",
+			httpresp.StatusCode, b.String())
```

Comment:
Does not dumping the entire response lose any important information, or is the status code the only important thing from it?

Reply:
I've read the vault [secret api](https://www.vaultproject.io/api-docs/secret/kv/kv-v2) (Error handling is not mentioned here) and the [api document](https://www.vaultproject.io/api-docs#error-response). In the [api document](https://www.vaultproject.io/api-docs#error-response) mentioned that the response body and `HTTP Status Codes` is used for error handing. So I think others are redundant information.

Reply:
Besides, I requested a key that does not exist, the comparison is in the related issue.

Reply:
Maybe the response body can be logged here at the debug level?

Reply:
Good! I'll fix that.